docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -7,14 +7,18 @@ description: "Learn about the User Feedback Widget configuration options."
 The User Feedback Widget offers many customization options, and if the available options are insufficient, you can [use your own UI](/platforms/react-native/user-feedback/#user-feedback-api).
 
 ![An image showing the main customization options for the User Feedback Widget](./img/mobile-user-feedback-widget-customization.png)
-To collect user feedback from inside your application use the `showFeedbackWidget` method.
+To collect user feedback from inside your application, use the `showFeedbackButton`/`hideFeedbackButton` to show/hide a button that triggers the Feedback Widget or the `showFeedbackWidget` method to present the widget directly.
 
 ```javascript
 import * as Sentry from "@sentry/react-native";
 
 Sentry.wrap(RootComponent);
 
 Sentry.showFeedbackWidget();
+
+Sentry.showFeedbackButton();
+
+Sentry.hideFeedbackButton();
 ```
```

Comment:
I think this would could be split into two code snippets, one as it was and one with the show/hide functions as if someone uses the button they won't have to call showWidget manually.

Reply:
````suggestion
```JavaScript {tabTitle:Widget}
import * as Sentry from "@sentry/react-native";

Sentry.wrap(RootComponent);

Sentry.showFeedbackWidget();
```

```JavaScript {tabTitle:Button}
import * as Sentry from "@sentry/react-native";

Sentry.wrap(RootComponent);

Sentry.showFeedbackButton();
Sentry.hideFeedbackButton();
```
````

Reply:
Good idea Krystof 🙇